pac4j-oidc/src/main/java/org/pac4j/oidc/config/OidcConfiguration.java

```diff
@@ -264,6 +264,9 @@ public void setResponseMode(final String responseMode) {
     }
 
     public String getLogoutUrl() {
+        if(logoutUrl == null && getProviderMetadata().getEndSessionEndpointURI() != null) {
+            return getProviderMetadata().getEndSessionEndpointURI().toString();
```

Comment:
In 2.1, we must remain backward compatible with 2.0, which is the case. Let's not go further and merge it as is.